src/CanvasRenderingContext2d.cc

```diff
@@ -2547,6 +2549,17 @@ NAN_SETTER(Context2d::SetFont) {
   context->state->fontDescription = sys_desc;
   pango_layout_set_font_description(context->_layout, sys_desc);
 
+  PangoAttribute *features;
+  if (strlen(*variant) > 0 && strcmp("small-caps", *variant)==0) {
+    features = pango_attr_font_features_new("smcp 1, onum 1");
+  }else{
```

Comment:
nit: could you put spaces before/after the brackets, and spaces before/after the `==` above

Reply:
gladly